spec/classes/web_spec.rb

```diff
@@ -13,17 +13,18 @@
   end
 
   on_supported_os.each do |os, facts|
-    next if facts[:os]['name'] == 'windows'
-    context "on #{os} " do
-      let :facts do
-        facts
-      end
-
-      if facts[:osfamily] == 'Archlinux' || facts[:osfamily] == 'Gentoo'
-        context 'with all defaults' do
-          it { is_expected.not_to compile }
+    %w[4.0 5.0].each do |zabbix_version|
+      next if facts[:os]['name'] == 'windows'
+      next if facts[:os]['name'] == 'Archlinux'
+      next if facts[:os]['name'] == 'Gentoo'
+      next if facts[:os]['name'] == 'Debian' and facts[:os]['release']['major'].to_i == 9
```

Comment:
I'd also avoid `.to_i` here
```suggestion
      next if facts[:os]['name'] == 'Debian' and facts[:os]['release']['major'] == '9'
```